parser/parser_test.go

```diff
@@ -1024,50 +1023,3 @@ func BenchmarkParse(b *testing.B) {
 	}
 	b.ReportAllocs()
 }
-
-var tableCompatible = []string{
-	`drop table IF EXISTS t;
-	CREATE TABLE t(c INT, index cidx (c));`,
-	`INSERT INTO t VALUES(1), (null), (2);`,
-	"SELECT COUNT(c) FROM t WHERE c IS NOT NULL;",
-	`select cast(null as char(30))`,
-	`select 0b01 + 1, 0b01000001 = "A"`,
-	"create table t (id tiny)",
-	"select        ((a+1))     from t",
-	"select !true from t",
-	"select IF(1>2,2,3) from t",
-	"select hex('TiDB') from t",
-	`show tables like 'show\_test'`,
-	`select _utf8"string";`,
-	"alter table show_test drop foreign key `fk`",
-	`insert t values('\x01')`,
-	`select "ab\_c"`,
-	`select "ab\%c"`,
-	`SELECT CONVERT("ABCD" USING ASCII);`,
-}
-
-func (s *testParserSuite) TestParserCompatible(c *C) {
```

Comment:
Not necessary.

The main purpose of those test cases is for easy debugging. when I found some statement can't work, I add them here, then fix the problem.

So all of those tests should be found some where else, maybe from tidb-test or other place.